lib/golomb/README.md

```diff
@@ -0,0 +1,26 @@
+# Golomb
+
+Golomb `lib/golomb` is a bcoin module intended to be used as a Golomb Coded Set
+for creating Compact Block Filters as specified in [BIP 158][0].  It is used to
+create block filters which can be used to quickly scan blocks for a given set
+of transaction outputs.
+
+Block filters are stored as an index using the blockstore.  To enable indexing
+filters, the config option `index-filter` should be enabled.  For mainnet, the
+indexing requires several hours on average and occupies ~4GB disk space.
```

Comment:
and occupies ~4GB of disk space as of height ...

Reply:
OK.